Gemfile: Switch from puppet to openvox

bastelfreak opened the pull request.

bastelfreak committed:
Gemfile: Switch from puppet to openvox

bastelfreak committed:
Gemfile: Switch from puppet to openvox

ekohl reviewed (commented):
Sorry for missing this. Is this all good or do we also need to update some gem versions? These aren't the latest versions:
https://github.com/theforeman/foreman-installer-modulesync/blob/e8e35b1a0b1d5ab88cdce5d19431347cb07e00d7/config_defaults.yml#L13-L34

bastelfreak committed:
Update gems to latest version

bastelfreak commented:
Tested in:
* https://github.com/theforeman/puppet-puppet/pull/999
* https://github.com/theforeman/puppet-puppetserver_foreman/pull/76

bastelfreak committed:
remove puppetlabs_spec_helper

This limits us to older puppet-lint versions and that conflicts with
latest voxpupuli-test releases.

bastelfreak committed:
puppet-lint-spaceship_operator_without_tag-check: Update to 2.0

bastelfreak committed:
switch from PSH to puppet_fixtures
